view/src/components/Profile.js

```diff
@@ -4,65 +4,40 @@ import { Link } from 'react-router-dom';
 import agent from '../agent';
 import ScheduleSelector from "react-schedule-selector";
 import { connect } from 'react-redux';
+import axios from 'axios';
```

Comment:
@diananova  are you using axios?

Reply:
Yep. It’s just less complicated.

Reply:
I see.. it will be cleaner if axios is imported in a different file like "agent" and call the function from there. 
Not necessarily for this PR right now tho